src/reporting/quicksight-custom-resource.ts

```diff
@@ -87,8 +87,8 @@ export function createQuicksightCustomResource(
 
   const eventViewColumns = `
     *, 
-    DATE_TRUNC('second', CONVERT_TIMEZONE('${props.timezone}', event_timestamp)) ::timestamp AS event_timestamp_local,
-    DATE_TRUNC('day', CONVERT_TIMEZONE('${props.timezone}', event_timestamp)) ::timestamp AS event_date
+    DATE_TRUNC('second', CONVERT_TIMEZONE('{{{timezone}}}', event_timestamp)) ::timestamp AS event_timestamp_local,
```

Comment:
Will it be replaced by QuickSight?

Reply:
no. this just used in dataset custom sql definition.  already checked.

Reply:
used in date set custom sql definition. already checked it is right.  We need check whether is it right at visual render time.